xl-objects.go

```diff
@@ -55,6 +92,12 @@ func newXLObjects(exportPaths ...string) (ObjectLayer, error) {
 	// cleaning up tmp files etc.
 	initObjectLayer(storage)
 
+	// Validate if format exists and input arguments are validated
+	// against backend format.
+	if err = isValidFormat(storage, exportPaths...); err != nil {
+		return nil, err
+	}
```

Comment:
isValidFormat should not initialize format.json if it did not exist. It should at the newXLObjects layer.